src/web/components/elements/UnsafeEmbedBlockComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { css } from 'emotion';
+
+type Props = {
+    html: string;
+    alt: string;
+};
+
+const fullWidthStyles = css`
+    width: 100%;
+`;
+export const UnsafeEmbedBlockComponent = ({ html, alt }: Props) => (
+    <iframe
+        className={`${fullWidthStyles} atom__iframe`}
+        title={alt}
+        data-cy="embed-block"
+        srcDoc={`<script src="https://interactive.guim.co.uk/libs/iframe-messenger/iframeMessenger.js"></script>
```

Comment:
This won't be initalised in the platform yet, see my comment around gu-script here: https://github.com/guardian/dotcom-rendering/pull/1946#discussion_r495889625